synchronization/src/cyface/java/de/cyface/synchronization/CyfaceAuthenticator.java

```diff
@@ -140,24 +138,12 @@ public Bundle getAuthToken(final @Nullable AccountAuthenticatorResponse response
         } catch (final MalformedURLException e) {
             sendErrorIntent(context, MALFORMED_URL.getCode(), e.getMessage());
             throw new NetworkErrorException(e);
-        } catch (final JSONException e) {
-            sendErrorIntent(context, UNREADABLE_HTTP_RESPONSE.getCode(), e.getMessage());
-            throw new NetworkErrorException(e);
         } catch (final SynchronisationException | RequestParsingException e) {
             sendErrorIntent(context, SYNCHRONIZATION_ERROR.getCode(), e.getMessage());
             throw new NetworkErrorException(e);
-        } catch (final DataTransmissionException e) {
-            sendErrorIntent(context, DATA_TRANSMISSION_ERROR.getCode(), e.getHttpStatusCode(), e.getMessage());
-            throw new NetworkErrorException(e);
-        } catch (final ResponseParsingException e) {
-            sendErrorIntent(context, UNREADABLE_HTTP_RESPONSE.getCode(), e.getMessage());
-            throw new NetworkErrorException(e);
         } catch (final UnauthorizedException e) {
             sendErrorIntent(context, UNAUTHORIZED.getCode(), e.getMessage());
             throw new NetworkErrorException(e);
-        } catch (final BadRequestException e) {
```

Comment:
Die Exception z.B. wird hard geworfen, weil das so nicht in der API Spezifikation auftaucht. Laut dieser wird beim Login nur:
200 oder 401 unauthorized zurückgegeben.

Reply:
**WICHTIG:**
Eventuell sollten wir allgemeine Server Fehler wie "400 Bad Request" oder "500 Internal Server Error" trotzdem soft abfangen und hoffen, dass Philipp die Server Logs regelmäßig auf so etwas prüft?